int8 number
---
bool high
bool result
